packages/angular-table/src/index.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Directive,
+  Input,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core'
+import {
+  RowData,
+  TableOptions,
+  TableOptionsResolved,
+  createTable,
+} from '@tanstack/table-core'
+
+export * from '@tanstack/table-core'
+
+@Directive({
+  selector: '[flexRender]',
+  standalone: true,
+})
+export class FlexRenderDirective implements OnInit {
+  private _flexRender: any
+
+  /** properties to render */
+  private _flexRenderProps: any
+
+  @Input({ required: true })
+  set flexRender(render: any) {
+    this._flexRender = render
+  }
+
+  @Input({ required: true })
+  set flexRenderProps(props: any) {
+    this._flexRenderProps = props
+  }
+
+  constructor(
+    private vcr: ViewContainerRef,
+    private templateRef: TemplateRef<any>
+  ) {}
+
+  ngOnInit(): void {
+    // This ensures that if the 'flexRender' input is set before the directive initializes,
+    // the component will be rendered when ngOnInit is called.
+    if (this._flexRender) {
+      this.renderComponent()
```

Comment:
This code overall LGTM, but I thought `ngOnInit` doesn't have access to `@Input` props - how does this code ever run?